test/boilerplate_suggestors/simple_props_and_state_class_migrator_test.dart

```diff
@@ -24,6 +27,12 @@ main() {
     final testSuggestor =
         getSuggestorTester(SimplePropsAndStateClassMigrator(converter));
 
+    setUpAll(() async {
+      semverHelper = SemverHelper(jsonDecode(
+          await File('test/boilerplate_suggestors/report.json')
```

Comment:
#nit as opposed to reading from a file, we could also make this a dart constant.

```dart
const reportJson = r'''{
  "exports": {
    ...
  }
}''';
```

```dart
semverHelper = SemverHelper(jsonDecode(reportJson));
```